AmplifyPlugins/Analytics/AWSPinpointAnalyticsPlugin/Dependency/Pinpoint/Analytics/AnalyticsClient.swift

```diff
@@ -0,0 +1,95 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import StoreKit
+
+class AnalyticsClient: InternalPinpointClient {
+    private let eventRecorder: EventRecorder
+    private var globalAttributes: [String: String] = [:]
+    private var globalMetrics: [String: Float] = [:]
```

Comment:
Can we use a more specific type here for the attributes and metrics keys? If it's not feasible feel free to ignore this